eng/Versions.props

```diff
@@ -81,7 +81,7 @@
   <PropertyGroup>
     <StaticCsVersion>0.2.0</StaticCsVersion>
     <!-- SDK dependencies -->
-    <MicrosoftDotNetApiCompatTaskVersion>8.0.100-preview.2.23107.1</MicrosoftDotNetApiCompatTaskVersion>
+    <MicrosoftDotNetApiCompatTaskVersion>8.0.100-preview.6.23301.17</MicrosoftDotNetApiCompatTaskVersion>
```

Comment:
Why don't we use the version that comes with the SDK? So then we don't have to manually update it.  I see it is 4 previews old right now.